README.md

```diff
@@ -14,17 +15,62 @@ $ curl -s https://getcomposer.org/installer | php
 $ php composer.phar install
 ```
 
-## Usage
+What's changed in version 6027
+------------------------------
 
-```
+## BC breaks listed
+
+ * Strict type hints have been added throughout. This may break some type assumptions made in earlier versions.
+ * In many classes Setters and Getters have been removed, the parameters have been moved to the class constructor
+ * Some classes are now `final` - use composition instead of inheritance
+
+What's changed in version 6025
+------------------------------
+
+## BC breaks listed
+
+ * The `grep` command and the `diff` command were removed
+
+## Changes
+
+ * The tests for integration testing the source files are split from the other tests
+ * Tests on travis use the build pipeline now
+
+Directory Structure
+-------------------
+
+* `bin` - Contains executable files
+* `build` - Contains various builds
+* `resources` - Files needed to build the various files, also used to validate the capabilities
+* `src` - The code of this project lives here
+* `tests` - The testing code of this project lives here
+
+the CLI commands
+----------------
+
+There is actually only one cli command available.
+
+## build
+
+This command is made to build a set defined of broswscap files.
```

Comment:
type-o in `browscap`  here.

Also the wording is a little awkward, maybe re-worded to something like this:

"This command is used to build a set of defined browscap files."

Reply:
changed